packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -40,9 +42,20 @@ export function useSliderThumb(
     isDisabled,
     validationState,
     trackRef,
-    inputRef
+    inputRef,
+    'aria-label': ariaLabel
   } = opts;
 
+  // Provide minimum and maximum aria-label when there are more than one slider thumb.
+  let formatMessage = useMessageFormatter(intlMessages);
+  if (state.values.length === 2 && !ariaLabel) {
+    if (index === 0) {
+      ariaLabel = formatMessage('minimum');
+    } else if (index === 1) {
+      ariaLabel = formatMessage('maximum');
+    }
+  }
```

Comment:
I'm also not sure if we should put this here as it kinda assumes that all two-thumbed sliders are range sliders. IMO the labels should go in Spectrum RangeSlider since we don't have a range specific aria hook.